src/lib/operators/join_hash.cpp

```diff
@@ -440,6 +453,15 @@ class JoinHash::JoinHashImpl : public AbstractReadOnlyOperatorImpl {
     }
     _performance.set_step_runtime(OperatorSteps::Building, timer_hash_map_building.lap());
 
+    // Store the element counts of the built hash tables. Depending on the Bloom filter, we might have significantly
+    // less values stored than in the initial input table.
+    for (const auto& hash_table : hash_tables) {
+      if (!hash_table) continue;
+
+      _performance.hash_table_distinct_value_count += hash_table->get_distinct_value_count();
+      _performance.hash_table_position_count += hash_table->get_position_count();
```

Comment:
I don't entirely recall the `get_...` naming discussion. I believe we decided against `get_` unless it sounds weird (e.g., `chunk` instead of `get_chunk`).